### PR TITLE
Box readme

### PR DESCRIPTION
**Pull request by mdittmer**
@kgrgreer PTAL in general. @adamvy PTAL in general + there's a section I'd like you to write ^_^

### CODE REVIEW

File: src/foam/box/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+```
+interface BoxRegistry {
+  Box  register(opt_name, opt_clientServerService, box);
+  void unregister(nameOrBox);
+}
+```
+
+The `register` method is used to register the input `box` under the name
+`opt_name`. If `opt_name` is omitted, the registry selects an available
```
Comment (kgrgreer) [inline src/foam/box/README.md:57]: Would 'random' be better than 'available' or 'random available'?

File: src/foam/box/README.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+```
+interface StubFactory {
+  Class get(Class);
+}
+```
+
+The `get()` method returns a stub class for the input class. Stub class
+instances have a `delegate` property that should be set to a box that accepts
+RPC messages over the unerlying class's interface.
```
Comment (kgrgreer) [inline src/foam/box/README.md]: s/underlying/underlying

**Comment by kgrgreer** [comment]
Added a couple comments, but LGTM.

File: src/foam/box/README.md (hunk 1 above)
Comment (mdittmer) [reply src/foam/box/README.md:57]: The default implementation is not random. It uses a counter. https://github.com/foam-framework/foam2/blob/master/src/lib/box.js#L235

**Pull request merged**